core/src/main/scala/org/broadinstitute/dsde/rawls/serviceperimeter/ServicePerimeterService.scala

```diff
@@ -30,6 +30,17 @@ class ServicePerimeterService(dataSource: SlickDataSource, gcsDAO: GoogleService
     dataAccess.workspaceQuery.getWorkspacesInPerimeter(servicePerimeterName)
   }
 
+  /**
+    * In its own transaction, look up all of the Billing Projects that use the specified
+    * ServicePerimeterName
+    *
+    * @param servicePerimeterName
+    * @return
+    */
+  private def collectBillingProjectsInPerimeter(servicePerimeterName: ServicePerimeterName, dataAccess: DataAccess): ReadAction[Seq[RawlsBillingProject]] = {
+    dataAccess.rawlsBillingProjectQuery.listProjectsWithServicePerimeterAndStatus(servicePerimeterName, CreationStatuses.Ready)
```

Comment:
this makes sense to me but just checking that we don't care about billing projects that are not in the `Ready` creation state?

Reply:
I believe so.

The statuses it could have are:
```
  case object Creating extends CreationStatus
  case object Ready extends CreationStatus
  case object Error extends CreationStatus
  case object AddingToPerimeter extends CreationStatus
```
If it's in **creating** status, its google project will be added to the perimeter when a workspace is created in the new billing project. Until a workspace is added and gets used, the google project will be empty. Since this is going out with PPW, workspaces won't use a v1 billing project's google project anyway. So for all those reasons I think we can leave this one out.

**Error** status - won't / can't be used anyway and its google project should be empty.
**AddingToPerimeter** status - this status isn't used anymore. We can probably remove this -- created a followup ticket in the post-ppw cleanup epic: https://broadworkbench.atlassian.net/browse/CA-1502

Reply:
FWIW, the Billing Project creation status field will be removed entirely as part of the PPW cleanup (or maybe even migration?)